package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@blackbaud/skyux-lib-restricted-view",
+  "version": "1.0.0-rc.1",
+  "description": "Single-page-application for skyux-lib-restricted-view",
+  "main": "bundles/bundle.umd.js",
+  "module": "index.ts",
+  "engines": {
+    "node": ">=6.11.1"
+  },
+  "scripts": {
+    "skyux": "skyux",
+    "test": "skyux test && skyux e2e && skyux build && npm run build",
```

Comment:
These commands are outdated (I created a PR to update them on the template), see:
https://github.com/blackbaud/skyux-sdk-template-library/pull/4/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2